demo/src/components/CombinedFilters.js

```diff
@@ -13,8 +13,8 @@ const CombinedFilters = () => {
     return (
         <CombinedFacets
             transform={transform}
-            RangeFacetItemComponent={RangeFacetItemComponent}
-            TextFacetItemComponent={TextFacetItemComponent}
+            RangeFacetItemComponent={<RangeFacetItemComponent />}
+            TextFacetItemComponent={<TextFacetItemComponent />}
```

Comment:
So we were accepting components before. Now that we are planning to make a switch to instances. Should we be also changing the way we are handling the instance.

Reply:
done